### PR TITLE
Faculty create critique ac

### PR DESCRIPTION
**Pull request by dbnorth**

### CODE REVIEW

**Review by JuliaFlores21 — CHANGES_REQUESTED** [review]
I think we should keep consistency and add a cancel button when faculty is adding a critique.

**Review by JuliaFlores21 — APPROVED** [review]
good

File: src/components/faculty/critique/FacultyCreateCritiqueDialog.vue [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+              <div class="font-weight-bold text-maroon text-h6 ml-1 mb-1">
+                Critique
+              </div>
+              <v-spacer></v-spacer>
+              <v-btn
+                flat
+                class="font-weight-semi-bold mt-0 mr-2 text-none text-white bg-blue flatChipBorder"
+                @click="clearCritique()"
+              >
+                Clear
```
[the remark this answers is not in the document]
Comment (dbnorth) [reply src/components/faculty/critique/FacultyCreateCritiqueDialog.vue]: fixed

File: src/views/BaseView.vue [hunk 2]
```diff
@@ -48,6 +49,9 @@ const fullPath = computed(() => route.path);
           v-if="fullPath === '/facultyHome' || fullPath === '/accompanistHome'"
         >
         </FacultyHome>
+        <FacultyCreateCritique
+          v-if="this.$route.path === '/facultyCreateCritique'"
```
[the remark this answers is not in the document]
Comment (dbnorth) [reply src/views/BaseView.vue]: fixed

**Pull request merged**